<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="jenkins-agent-10-25-220-110" name="com.mypurecloud.sdk.v2.ApiClientRetryTest" tests="12" failures="0" timestamp="25 Nov 2021 07:36:29 GMT" time="70.119" errors="0">
  <testcase name="invokeTestWith_502" time="13.051" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_200" time="0.017" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_502" time="1.002" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_504" time="2.055" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_429" time="6.238" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_429_And_No_MaxRetryTime" time="0.069" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_429" time="1.004" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_503_And_RetryConfig" time="3.006" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_504_And_No_RetryAfter_Header" time="3.003" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_504_No_MaxRetryTime" time="0.103" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_502_And_0_MaxRetryTime" time="0.497" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_503" time="40.074" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryTest -->
